netcat-openbsd.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: netcat-openbsd
-  version: "1.226"
```

Comment:
Are there any other consequences from changing this versioning scheme? release-monitoring.org doesn't have the -1.1 suffix but the repo does, wasn't sure which to use.